core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/bundle/Bundle.java

```diff
@@ -55,7 +55,7 @@ public String getResourceType() {
 		return RESOURCE_TYPE;
 	}
 
-	public BundleCreateRequestBuilder toCreateRequest() {
+	public BundleCreateRequestBuilder toCreateRequestBuilder() {
```

Comment:
I think we have deliberately named these without the Builder suffix even if the returned object is a builder. This creates a bit more fluent and readable API than including the Builder part. I'm open to suggestions.

Reply:
Hmm, I was hesitant with this given if I see `toCreateRequest` I personally would assume I have a request ready to go. 
However, it is a builder where I can change basically anything before building it and sending it away.

I'm fine with both ways though, let's settle on one of them and I'm happy to change the rest. Previously it was not consistent for CodeSystem, Bundle, ValueSet and ConceptMap.